passes/techmap/muxcover.cc

```diff
@@ -407,21 +410,29 @@ struct MuxcoverWorker
 			ok = ok && follow_muxtree(S8, tree, bit, "BBBS");
 
 			if (nodecode)
-				ok = ok && S1 == S2 && S2 == S3 && S3 == S4 && S4 == S5 && S5 == S6 && S6 == S7 && S7 == S8;
+				ok = ok && (S1 == S2 || S1 == State::Sx || S2 == State::Sx) &&
+					/*S2 == S3 &&*/
+					(S3 == S4 || S3 == State::Sx || S4 == State::Sx) &&
+					/*S4 == S5 &&*/
+					(S5 == S6 || S5 == State::Sx || S6 == State::Sx) &&
+					/*S6 == S7 &&*/
+					(S7 == S8 || S7 == State::Sx || S8 == State::Sx);
 
 			ok = ok && follow_muxtree(T1, tree, bit, "AAS");
 			ok = ok && follow_muxtree(T2, tree, bit, "ABS");
 			ok = ok && follow_muxtree(T3, tree, bit, "BAS");
 			ok = ok && follow_muxtree(T4, tree, bit, "BBS");
 
 			if (nodecode)
-				ok = ok && T1 == T2 && T2 == T3 && T3 == T4;
+				ok = ok && (T1 == T2 || T1 == State::Sx || T2 == State::Sx) &&
+					/*T2 == T3 &&*/
```

Comment:
My understanding is that this check is overly strict since `U1 == U2` is checked downstream?